Marquee/ViewControllers/ViewController.swift

```diff
@@ -177,14 +157,15 @@ extension ViewController:UITableViewDelegate, UITableViewDataSource, collectionV
             }
             fetchFavorites()
         } else {
+            print("trying to create favorite")
             let newFavorite = Favorites(context: context)
             
             newFavorite.title = selectedContent.title
             newFavorite.overview = selectedContent.overview
-            newFavorite.poster_path = selectedContent.poster_path
-            newFavorite.release_date = selectedContent.release_date
-            newFavorite.vote_average = selectedContent.vote_average
-            
+            newFavorite.posterPath = selectedContent.posterPath
+            newFavorite.releaseDate = selectedContent.releaseDate
+            newFavorite.voteAverage = selectedContent.voteAverage
+            print(newFavorite)
```

Comment:
Print statements are a poor mans way of logging lol ... might want to consider removing print statement for production code and just keep them for debugging. Also using Apple's built in logging system might not be a bad idea.